release/gvfs.xml

```diff
@@ -28,6 +28,9 @@
   <option id="metadata">
     <label>GVFS Metadata</label>
     <description>Delete metadata about previously mounted filesystems</description>
-    <action command="delete" search="glob" path="~/.local/share/gvfs-metadata/*"/>
+    <warning>All customizations like comments, emblems, icons and any position information of icons and within documents will be deleted as well.</warning>
+    <action command="delete" search="glob" path="${XDG_DATA_HOME}/gvfs-metadata/*"/>      
+    <!-- Remaining data pre XDG adaption -->
+    <action command="delete" search="glob" path="${HOME}/.local/share/gvfs-metadata/*"/>
```

Comment:
BleachBit automatically defines XDG_DATA_HOME if it is not defined, so it's not necessary to have line 34.  (See the [documentation about variables](https://www.bleachbit.org/help).) Please remove the unneeded line. One problem is that it will cause duplicates in the preview, which inflates the estimate of how much will be cleaned.